src/utils/aeternity/client.js

```diff
@@ -20,8 +20,8 @@ export const aeClient = reactive({
  * This client can not sign transactions that require funds (everything except static contract calls)
  * @returns {Promise<boolean>}
  */
-export const aeInitClient = async () => {
-  const { isConnected, isConnecting, isStatic } = toRefs(aeClient)
+export const initClient = async () => {
```

Comment:
I have removed these prefixing as I saw no benefit in it, only bloating code and worse readiness